Automattic/jetpack
# Search: Fix layout issue in Search dashboard when Hello Dolly is active.

Fixes #24940 

Before:

![177076460-0b578419-b5c9-4827-b307-3ab34975a7bd](https://user-images.githubusercontent.com/40267301/179511522-4ef3c37c-ad6d-4f4f-940c-4bd515ff7532.png)

After:

![UpdatedLayout](https://user-images.githubusercontent.com/40267301/179511705-055c67f9-8a34-4f6d-a3ed-9cffa9437561.png)


#### Changes proposed in this Pull Request:


Removes the `display: flex` from the top-level dashboard class. This is applied as needed to inner classes so it shouldn't be needed at the top level.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* On any site with Jetpack or Jetpack Search installed, activate the Hello Dolly plugin.
* Confirm the layout looks like the 2nd screenshot above.
* Try with and without a Jetpack connection.
* Try with and without a plan.
